spec/granite_spec.cr

```diff
@@ -2,7 +2,7 @@ require "./spec_helper"
 
 describe "Granite::Base" do
   describe "JSON" do
-    context ".from_json" do
+    describe "#from_json" do
```

Comment:
This should be `.from_json` because it is a class method but not instance method.

Reply:
Yes, yes it should.